### PR TITLE
Jumpstart exercises

### PR DESCRIPTION
**Pull request by CYJW**
## Jumpstart Modules
Which week are you submitting completed practice exercises for? 

- [ ] Week 1 (Models)
- [ ] Week 2 (Tests, Docs, Sources)
- [X] Week 3 (Jinja, Macros, Packages)
- [ ] Week 4 (Materializations, Analyses, Seeds)
## Focus the feedback
What models/macros/files would you like feedback on?  Be specific.
Hi Amy,
I didn't do the packages exercise. (I'm not in a Developer role at BCG Data Factory
Justine

### CODE REVIEW

**Comment by CYJW** [comment]
@amychen1776 
Hope you had a good weekend.
Please take a look when you have a chance
Justine

File: models/marts/core/int_orders__pivot.sql [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{%- set payment_methods = ['bank_transfer','coupon','credit_card','gift_card'] -%}
+
+with payments as (
+    select * from {{ ref('stg_payments') }}
+),
+
+pivoted as (
+    select
+        order_id,
+        {% for p in payment_methods -%}
```
Comment (amychen1776) [inline models/marts/core/int_orders__pivot.sql:10]: Using `p` is acceptable here! In the future, I recommend having something more descriptive. It makes your code more readable for those who might be hoping into the project. This goes in line with why we never alias field names to something shorten like `price as p`. Code easily readable to humans will save everyone some brainpower :)

File: models/marts/core/int_orders__pivot.sql [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        {% endfor -%}
+
+    from payments
+    where status = 'success'
+    group by 1
+        )
+)
+
+
+sum(case when payment_method = 'bank_transfer' then amount else 0 end) as bank_transfer_payment,
```
Comment (amychen1776) [inline models/marts/core/int_orders__pivot.sql]: I think you meant to erase this and just have a `select * from pivoted`

File: models/staging/stripe/stg_payments.sql [hunk 3]
```diff
@@ -3,7 +3,8 @@ SELECT
     ORDERID AS ORDER_ID,
     PAYMENTMETHOD AS PAYMENT_METHOD,
     STATUS,
-    AMOUNT,
+    AMOUNT AS RAW_AMOUNT,
```
Comment (amychen1776) [inline models/staging/stripe/stg_payments.sql]: Since the rest of your project is in lowercase, I recommend updating this model as well.

**Review by amychen1776 — CHANGES_REQUESTED** [review]
Overall, this looks great. Just a few alterations and you're ready to go!

**Comment by amychen1776** [comment]
@CYJW I was curious why you weren't able to install a package? The dbt_utils package should be accessible to you and we recommend installing it on every dbt project.

**Comment by amychen1776** [comment]
I did also want to provide a suggestion about your branch name. We are strong believers in using feature branches which means that when we start a new piece of work, we create a new branch name that is descriptive of the work we will be doing. I recommend having branches like `feature/week_3_exercises` and deleting them after you have merged in your pull request. It's a good habit to get into as you start to dive into your own dbt project.

**Comment by CYJW** [comment]
> @CYJW I was curious why you weren't able to install a package? The dbt_utils package should be accessible to you and we recommend installing it on every dbt project.

@amychen1776 
I got this error: when I ran dbt deps
I ran out of time and I'm not sure that I will need utils (since I'm a reader not a coder in DBT - I'm the Product Owner, not a hands on developer)

SummaryDetails
Installing fishtown-analytics/dbt_utils@0.6.2
  Installed from version 0.6.2
Runtime Error
  Failed to read package at /usr/src/develop/user-10031/environment-12269/repository-10072/dbt_modules/dbt_utils: Runtime Error
    This version of dbt is not supported with the 'dbt_utils' package.
      Installed version of dbt: =0.17.2
      Required version of dbt for 'dbt_utils': ['>=0.18.0', '<0.19.0']
    Check the requirements for the 'dbt_utils' package, or run dbt again with --no-version-check

File: models/marts/core/int_orders__pivot.sql (hunk 2 above)
Comment (CYJW) [reply models/marts/core/int_orders__pivot.sql]: @amychen1776 

Done thanks

File: models/staging/stripe/stg_payments.sql (hunk 3 above)
Comment (CYJW) [reply models/staging/stripe/stg_payments.sql]: @amychen1776 
Committed new version with lower case

**Pull request merged**